indexer/migrations/20230523_create_schema.sql

```diff
@@ -0,0 +1,99 @@
+/**
+ * BLOCK DATA
+ */
```

Comment:
Should we have a table basically indexing bridges?   So it would keep track of dai bridges, snx bridges, and standard bridges?

Could imagine wanting to index our ERC721 bridge in future too